doc/INSTALL.MD

```diff
@@ -0,0 +1,46 @@
+# Install Travel Support Program Application
+
+All the Information you need to Install TSP. If you face any issues with installing don't hesitate to <a href="https://github.com/openSUSE/travel-support-program#contact">contact us</a>.
+
+## Getting Started
+
+As most regular Ruby on Rails applications:
+
+```
+$ git clone https://github.com/openSUSE/travel-support-program
+$ cd travel-support-program
+$ vi Gemfile # edit Gemfile to uncomment your favorite database driver
+$ bundle
+$ cp config/database.example.yml config/database.yml
+$ cp config/site.example.yml config/site.yml
+$ vi config/site.yml # edit configuration DON'T FORGET to set proper secrets
+$ rake db:create db:migrate db:seed
+$ rails s
+```
+
+You can access the application pointing your browser to <http://localhost:3000/>
+Four initial users should be available with the following emails and passwords:
+
+* tspmember@example.com / tspmember1
+* requester@example.com / requester1
+* administrative@example.com / administrative1
+* assistant@example.com / assistant1
+
+## Sending Emails in Travel Support Program
+
+To send emails with travel support program you need to setup some external mail service. For example using the gmail smtp, just add the smtp settings using `config.action_mailer.smtp_settings` in the `environment` files.
+For gmail smtp settings you can checkout this <a href="https://support.google.com/a/answer/176600?hl=en">link</a>.
+
+Their are some email variables provided in the site.yml file which your can use to configure emails in TSP :
+
+| Variable          | Content           | Purpose               |
+|----------         |---------          |---------                  |
+| async_emails      | true              | To use delayed_jobs for emails         |
```

Comment:
Please either align all of the 4th | chars or remove the whitespaces before 'To use delayed_jobs for emails'